tests/test_peer_did_utils.py

```diff
@@ -50,3 +53,88 @@ def test_encode_service_with_multiple_entries_list():
         "nNvbWVtZWRpYXRvciNzb21la2V5Il19LHsidCI6ImRtIiwicyI6Imh0dHBzOi8vZXhhbXBsZS5jb20vZW5kcG9p"
         "bnQyIiwiciI6WyJkaWQ6ZXhhbXBsZTpzb21lbWVkaWF0b3Ijc29tZWtleTIiXX1d"
     )
+
+
+PEER_DID = "did:peer:0z6MkqRYqQiSgvZQdnBytw86Qbs2ZWUkGv22od935YF4s8M7V"
+
+
+@pytest.mark.parametrize(
+    "test_value",
+    [
+        (
+            "6MkqRYqQiSgvZQdnBytw86Qbs2ZWUkGv22od935YF4s8M7V",
```

Comment:
I think makes sense to put some comment regarding the parameters (why two, what is the difference)

Reply:
Refactored

Reply:
for the moment the list of these tests is dumped as:
```bash
$ pytest --collect-only -k test_decode_encumbasis
============================================================================================================ test session starts ============================================================================================================
platform linux -- Python 3.6.13, pytest-5.4.3, py-1.10.0, pluggy-0.13.1
rootdir: /home/and/Projects/GitHub/all/sicpa-dlab/peer-did-python
collected 68 items / 62 deselected / 6 selected                                                                                                                                                                                             
<Package /home/and/Projects/GitHub/all/sicpa-dlab/peer-did-python/tests>
  <Module test_peer_did_utils.py>
    <Function test_decode_encumbasis[z6MkqRYqQiSgvZQdnBytw86Qbs2ZWUkGv22od935YF4s8M7V-DIDDocVerMaterialFormat.BASE58-expected0]>
    <Function test_decode_encumbasis[z6LSbysY2xFMRpGMhb7tFTLMpeuPRaqaWM1yECx2AtzE3KCc-DIDDocVerMaterialFormat.BASE58-expected1]>
    <Function test_decode_encumbasis[z6MkqRYqQiSgvZQdnBytw86Qbs2ZWUkGv22od935YF4s8M7V-DIDDocVerMaterialFormat.MULTIBASE-expected2]>
    <Function test_decode_encumbasis[z6LSbysY2xFMRpGMhb7tFTLMpeuPRaqaWM1yECx2AtzE3KCc-DIDDocVerMaterialFormat.MULTIBASE-expected3]>
    <Function test_decode_encumbasis[z6MkqRYqQiSgvZQdnBytw86Qbs2ZWUkGv22od935YF4s8M7V-DIDDocVerMaterialFormat.JWK-expected4]>
    <Function test_decode_encumbasis[z6LSbysY2xFMRpGMhb7tFTLMpeuPRaqaWM1yECx2AtzE3KCc-DIDDocVerMaterialFormat.JWK-expected5]>
```

I suggest to consider to use [pytest.param](https://docs.pytest.org/en/6.2.x/example/parametrize.html#different-options-for-test-ids) instead, e.g.:
```python
        pytest.param(
            "z6MkqRYqQiSgvZQdnBytw86Qbs2ZWUkGv22od935YF4s8M7V",
            DIDDocVerMaterialFormat.BASE58,
            VerificationMaterial(

                value="ByHnpUCFb1vAfh9CFZ8ZkmUZguURW8nSw889hy6rD8L7",
                encnumbasis="6MkqRYqQiSgvZQdnBytw86Qbs2ZWUkGv22od935YF4s8M7V",
            ),
            id="base58-Ed25519VerificationKey2018"
         )
```